tests/h/factories_test.py

```diff
@@ -15,78 +15,30 @@ def test_id(self):
         annotation_1 = factories.Annotation()
         annotation_2 = factories.Annotation()
 
-        assert annotation_1.get("id")
-        assert annotation_2.get("id")
-        assert annotation_1["id"] != annotation_2["id"]
+        assert annotation_1.id
+        assert annotation_2.id
+        assert annotation_1.id != annotation_2.id
 
     def test_text(self):
         """Each annotation should have unique note text."""
         annotation_1 = factories.Annotation()
         annotation_2 = factories.Annotation()
 
-        assert annotation_1.get("text")
-        assert annotation_2.get("text")
-        assert annotation_1["text"] != annotation_2["text"]
+        assert annotation_1.text
+        assert annotation_2.text
+        assert annotation_1.text != annotation_2.text
 
-    def test_custom_user(self):
-        """A custom username should be used in the user field."""
-        annotation = factories.Annotation(username="bobo")
-        assert "bobo" in annotation["user"]
-        assert "username" not in annotation
-
-    def test_created_date(self):
-        """Annotations should have a created date from the current time."""
-        before = datetime.datetime.now()
-
-        annotation = factories.Annotation()
-
-        after = datetime.datetime.now()
-        created = datetime.datetime.strptime(
-            annotation["created"], "%Y-%m-%dT%H:%M:%S.%f")
-
-        assert before < created < after
-
-    def test_updated_date(self):
-        """Annotations should have an updated date from the current time."""
-        before = datetime.datetime.now()
-
-        annotation = factories.Annotation()
-
-        after = datetime.datetime.now()
-        updated = datetime.datetime.strptime(
-            annotation["updated"], "%Y-%m-%dT%H:%M:%S.%f")
-
-        assert before < updated < after
-
-    def test_tags(self):
-        """It should be possible to choose the number of tags with num_tags."""
-        # If num_tags isn't passed the factory chooses a random number of tags.
-        # Here we choose a num_tags higher than the upper range of this random
-        # choice, so there's no chance of random false positive test passes.
-        annotation = factories.Annotation(num_tags=20)
-        assert len(annotation["tags"]) == 20
-        assert "num_tags" not in annotation
```

Comment:
This ability to specify the number of tags was removed. You can do `factories.Annotation(tags=['foo', 'bar'])` though.